x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/persistence/TrainedModelProvider.java

```diff
@@ -1086,32 +1087,36 @@ private static QueryBuilder buildQueryIdExpressionQuery(String[] tokens, String
         return results;
     }
 
-    private static String getDefinitionFromDocs(List<TrainedModelDefinitionDoc> docs, String modelId) throws ElasticsearchException {
-        String compressedString = docs.stream()
-            .map(TrainedModelDefinitionDoc::getCompressedString)
-            .collect(Collectors.joining());
-        // BWC for when we tracked the total definition length
```

Comment:
total definition length is now tracked again as we need to know the size of PyTorch models up front